app/javascript/components/Notifications/index.js

```diff
@@ -0,0 +1,37 @@
+import React, { useContext } from 'react'
+
+import * as R from 'ramda'
+import { Grid, Row, Col } from '@zendeskgarden/react-grid'
+import { ThemeProvider } from '@zendeskgarden/react-theming'
+import { UserContext } from '../../context'
+
+import ConfirmProfileAlert from './ConfirmProfileAlert'
+
+import '@zendeskgarden/react-grid/dist/styles.css'
+import '@zendeskgarden/react-notifications/dist/styles.css'
+
+const isFirstSignIn = R.propSatisfies(R.isNil, 'lastSignInAt')
```

Comment:
is there something we can do so user's that've already sign in before and have already changed their office won't get this prompt? It _may_ be noisy for those who've already set their office.

Reply:
hmm, I can backfill the user preference value for all existing users